.github/workflows/php.yml

```diff
@@ -0,0 +1,64 @@
+name: CI FOSHttpCacheBundle
+
+env:
+  SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+on:
+  push:
+    branches: [ master, /^\d+\.\d+$/, hotfix/move-to-github-action ]
+  pull_request:
+    branches: [ master, /^\d+\.\d+$/, hotfix/move-to-github-action ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: ['7.2', '7.3']
+        composer-flag: ['--prefer-stable --prefer-lowest']
```

Comment:
this should go away, we only want to have the 2 specific builds to prefer-lowest